src/NHibernate/Logging.cs

```diff
@@ -52,6 +52,10 @@ public static class NHibernateLogger
 		private const string nhibernateLoggerConfKey = "nhibernate-logger";
 		private static INHibernateLoggerFactory _loggerFactory;
 
+#pragma warning disable 618
+		internal static ILoggerFactory LegacyLoggerFactory { private set; get; }
```

Comment:
A bit strange to see the setter before the getter. I would rather have it in the usual order.